packages/devtools_app/lib/src/flutter/theme.dart

```diff
@@ -61,3 +63,5 @@ const devtoolsBlue = ColorSwatch<int>(600, {
   600: Color(0xFF0175C2),
   400: Color(0xFF13B9FD),
 });
+
+final chartBackgroundColor = ThemedColor(Colors.grey[50], Colors.grey[850]);
```

Comment:
Should the chart background be gray for light theme?

Reply:
grey[50] is the same color as the app background for light theme. This eliminates any border around the chart that would be visible if the chart background were a different color than the app background.